src/app/core/basic-datatypes/string/edit-url/edit-url.component.html

```diff
@@ -0,0 +1,17 @@
+<mat-form-field>
+  <mat-label>{{ label }}</mat-label>
+
+  <ng-container *ngIf="formControl.disabled; else inputField">
+    <a [href]="formControl.value" target="_blank" matInput>
+      {{ formControl.value || "No URL available" }}
+    </a>
+  </ng-container>
+  <ng-template #inputField>
+    <input [formControl]="formControl" matInput [title]="label" type="url" />
+  </ng-template>
+
+  <mat-error *ngIf="formControl.invalid && formControl.touched">
+    <span *ngIf="formControl.errors?.['pattern']">Invalid URL format</span>
```

Comment:
```suggestion
    <span *ngIf="formControl.errors?.['pattern']" i18n>Invalid URL format</span>
```
marking this to be translatable

Reply:
this i18 tag is still missing.

Also, is "pattern" used anywhere to check validity?